wf/entrypoint.py

```diff
@@ -0,0 +1,430 @@
+        *get_flag("tax_agglom_min", tax_agglom_min),
+        *get_flag("tax_agglom_max", tax_agglom_max),
+        *get_flag("ancom_sample_min_count", ancom_sample_min_count),
+        *get_flag("ancom", ancom),
+        *get_flag("ancombc", ancombc),
+        *get_flag("ancombc_formula", ancombc_formula),
+        *get_flag("ancombc_formula_reflvl", ancombc_formula_reflvl),
+        *get_flag("ancombc_effect_size", ancombc_effect_size),
+        *get_flag("ancombc_significance", f"{ancombc_significance:.6f}" if ancombc_significance is not None else None),
+        *get_flag("report_template", report_template),
+        *get_flag("report_css", report_css),
+        *get_flag("report_logo", report_logo),
+        *get_flag("report_title", report_title),
+        *get_flag("report_abstract", report_abstract),
+        *get_flag("skip_fastqc", skip_fastqc),
+        *get_flag("skip_cutadapt", skip_cutadapt),
+        *get_flag("skip_dada_quality", skip_dada_quality),
+        *get_flag("skip_barrnap", skip_barrnap),
+        *get_flag("skip_qiime", skip_qiime),
+        *get_flag("skip_qiime_downstream", skip_qiime_downstream),
+        *get_flag("skip_taxonomy", skip_taxonomy),
+        *get_flag("skip_dada_taxonomy", skip_dada_taxonomy),
+        *get_flag("skip_dada_addspecies", skip_dada_addspecies),
+        *get_flag("skip_barplot", skip_barplot),
+        *get_flag("skip_abundance_tables", skip_abundance_tables),
+        *get_flag("skip_alpha_rarefaction", skip_alpha_rarefaction),
+        *get_flag("skip_diversity_indices", skip_diversity_indices),
+        *get_flag("skip_multiqc", skip_multiqc),
+        *get_flag("skip_report", skip_report),
+        *get_flag("seed", seed),
+        *get_flag("multiqc_methods_description", multiqc_methods_description),
+        *get_flag("max_cpus", max_cpus),
+        *get_flag("max_memory", max_memory),
+        *get_flag("max_time", max_time),
+    ]
+
+    print("Launching Nextflow Runtime")
+    print(" ".join(cmd))
+    print(flush=True)
+
+    failed = False
+    try:
+        env = {
+            **os.environ,
+            "NXF_ANSI_LOG": "false",
+            "NXF_HOME": "/root/.nextflow",
+            "NXF_OPTS": "-Xms5g -Xmx30g -XX:ActiveProcessorCount=8",
+            "NXF_DISABLE_CHECK_LATEST": "true",
+            "NXF_ENABLE_VIRTUAL_THREADS": "false",
+        }
+        subprocess.run(
+            cmd,
+            env=env,
+            check=True,
+            cwd=str(shared_dir),
+        )
+    except subprocess.CalledProcessError:
+        failed = True
+    finally:
+        print()
+
+        nextflow_log = shared_dir / ".nextflow.log"
+        if nextflow_log.exists():
+            name = _get_execution_name()
+            if name is None:
+                print("Skipping logs upload, failed to get execution name")
+            else:
+                remote = LPath(urljoins("latch:///your_log_dir/nf_nf_core_ampliseq", name, "nextflow.log"))
```

Comment:
your_log_dir should maybe be changed to nf_core_ampliseq_logs ? 

could also put the log file in the directory generated by the run name if you think that makes more sense

Reply:
I prefered a single log dir with subfolders for every nfcore pipeline that was run
Inside those are another layer of subfolders with the run names and those contain logs

<img width="836" alt="image" src="https://github.com/user-attachments/assets/f2df87e7-19d8-49db-9651-66401e803abf">